moveit_ros/moveit_servo/include/moveit_servo/servo.hpp

```diff
@@ -232,6 +230,9 @@ class Servo
 
   // Map between joint subgroup names and corresponding joint name - move group indices map
   std::unordered_map<std::string, JointNameToMoveGroupIndexMap> joint_name_to_index_maps_;
+
+  // joint limit acceleration for most restricted joint
+  double joint_acceleration_limit = 0.0;
```

Comment:
I don't think we need this variable, will comment in the part where this is calculated.

Reply:
Removed